tests/www/approvals_views/test_list.py

```diff
@@ -214,3 +216,24 @@ def test_approval_state_filters(self, client):
             </span>""",
             html=True,
         )
+
+        # Check IAE pass remainder days
+        assertContains(
+            response,
+            """365 jours""",
+        )
```

Comment:
```suggestion
        assertContains(response, "365 jours")
```
(et pareil pour les suivants)